css/style.css

```diff
@@ -131,3 +135,37 @@ button {
 /* card-content/end */
 
 /* work/end */
+
+/* footer/start */
+
+.footer .wrapper {
```

Comment:
- [ ] You have done a great job with your contact form :fire: but there is an issue because the form is overflowing to the right side. I would suggest fixing this in order to match the Figma design :raised_hands:

  Your design:
  ![image](https://user-images.githubusercontent.com/81522435/140526102-ad2eae53-857b-42ba-a41e-bb23a9e5a8eb.png)